content/pages/disability-benefits/apply/claim-types/fully-developed-claim-alt.md

```diff
@@ -0,0 +1,116 @@
+2. When you file your claim, you'll need to turn in:
+    - A copy of your DD214 or other separation documents,  **and**
+    - Service treatment records,  **and**
+    - All private and VA medical records related to your illness or injury, **and**
+    - Information about any health records that are available at a federal facility, like a VA medical center or clinic
+3. Go to any VA medical appointments required to decide your claim.
+
+
+<br>
+
+#### Standard Claim Process
+
+With a standard claim, you’ll give us permission to gather evidence or records for you. You’ll need to give us enough information about the evidence so we can request it from the right person or agency.
+
+If a person or agency refuses to turn over the evidence, asks for money for the evidence, or can’t get the evidence, we’ll ask you to try to get the evidence or records. It’s your responsibility to make sure we receive all requested records that aren’t at a federal agency or office.
+
+</div>
+</li>
+<li>
+<button class="usa-button-unstyled usa-accordion-button" aria-controls="va-will-help">How will VA help me get evidence for my claim?</button>
+<div id="va-will-help" class="usa-accordion-content">
+
+#### FDC Program
+
+Under the FDC program, we’ll handle the evidence-gathering steps listed below. 
+
+**We’ll:**
+
+* Request relevant records that you identify and authorize us to get from a federal facility, like a VA medical center
+* Schedule a health exam or get a medical opinion for you, if we decide we need it for your claim
+
+<br>
+
+#### Standard Claim Process
+
+With the standard claim process, we’ll handle the evidence-gathering steps listed below.
+
+**We'll:**
+
+* Request relevant records that you identify and authorize us to get from a federal facility, like a VA medical center
+* Schedule a health exam or get a medical opinion for you, if we decide we need it for your claim
+* Make every reasonable effort to request relevant records that you authorize us to get that aren’t at a federal facility. These may include records from state or local governments, documents from a private doctor or hospital, or records from current or former employers.
+
+</div>
+</li>
+<li>
+<button class="usa-button-unstyled usa-accordion-button" aria-controls="when-to-send">When should I turn in my evidence?</button>
+<div id="when-to-send" class="usa-accordion-content">
+
+#### FDC Program
+
+You’ll need to turn in the information and evidence at the same time as you file your claim.
+
+<div class="usa-alert usa-alert-warning no-background-image">
+
+**Note:** If you turn in additional information or evidence after you send in your fully developed claim, we’ll remove your claim from the FDC program and process it as a standard claim. If we decide your claim before 1 year from the date we receive the claim, you’ll have the rest of that 1-year period to turn in additional information or evidence to support your claim.
```

Comment:
Should "If we decide your claim before 1 year from the date we receive the claim" be something like "If we decide your claim within 1 year (from the date we receive the claim)"?